protobuf/go/flow/access/access_grpc.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-grpc v1.3.0
+// - protoc-gen-go-grpc v1.2.0
```

Comment:
looks like you generated with an older version than the last person to update these. can you update your client?